sdk/storage/Azure.Storage.Common/src/Shared/ClientsideEncryption/ClientSideEncryptionOptionsExtensions.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Cryptography
+{
+    internal static class ClientSideEncryptionOptionsExtensions
+    {
+        public static ClientSideEncryptionOptions Clone(this ClientSideEncryptionOptions options)
```

Comment:
Nit - maybe add a mainteance note in ClientSideEncryptionOptions that this msut be updated when changes are introduced there?